target/linux/ath79/mikrotik/target.mk

```diff
@@ -3,7 +3,7 @@ FEATURES += minor nand
 KERNELNAME := vmlinux vmlinuz
 IMAGES_DIR := ../../..
 
-DEFAULT_PACKAGES += wpad-basic-mbedtls
+DEFAULT_PACKAGES += wpad-basic-mbedtls yafut
```

Comment:
Why do you add `yafut` to all mikrotik devices and not only the nand devices like it iwas done with `nand-utils` before?

Reply:
I just read you added it here to have it in the ramdisk too.
Please add this to the commit description.

Reply:
Sure, thanks.  Does e322e5268948f8ff88b2246536a5e360fa0236b2 work for you?  (I applied the `mikrotik_nor` tweak you suggested and updated the second paragraph of the log message compared to commit 4a0bbce8471ebd0c15af43aa9f6ebd89c7f8d674.)

Reply:
looks good.